src/azul/collections.py

```diff
@@ -124,3 +127,25 @@ def compose_keys(f, g):
 
     """
     return lambda v: f(g(v))
+
+
+def none_safe_itemgetter(*items):
+    """
+    Return a `None` safe callable object that fetches the given item(s) from its
+    operand.
+
+    >>> f = none_safe_itemgetter('foo', 'bar')
+    >>> f({'foo': 'foo', 'bar': 'bar'})
+    ('foo', 'bar')
+
+    >>> f(None)
+    (None, None)
+
+    """
```

Comment:
```suggestion
    >>> none_safe_itemgetter('foo')({'foo':123})
    123

    >>> none_safe_itemgetter('foo')(None)
    None
    """
```

The current implementation does not handle this.